pkg/infoschema/tables.go

```diff
@@ -245,20 +240,20 @@ const (
 )
 
 var tableIDMap = map[string]int64{
-	TableSchemata:                           autoid.InformationSchemaDBID + 1,
-	TableTables:                             autoid.InformationSchemaDBID + 2,
-	TableColumns:                            autoid.InformationSchemaDBID + 3,
-	tableColumnStatistics:                   autoid.InformationSchemaDBID + 4,
-	TableStatistics:                         autoid.InformationSchemaDBID + 5,
-	TableCharacterSets:                      autoid.InformationSchemaDBID + 6,
-	TableCollations:                         autoid.InformationSchemaDBID + 7,
-	tableFiles:                              autoid.InformationSchemaDBID + 8,
-	CatalogVal:                              autoid.InformationSchemaDBID + 9,
-	TableProfiling:                          autoid.InformationSchemaDBID + 10,
-	TablePartitions:                         autoid.InformationSchemaDBID + 11,
-	TableKeyColumn:                          autoid.InformationSchemaDBID + 12,
-	TableReferConst:                         autoid.InformationSchemaDBID + 13,
-	TableSessionVar:                         autoid.InformationSchemaDBID + 14,
+	TableSchemata:         autoid.InformationSchemaDBID + 1,
+	TableTables:           autoid.InformationSchemaDBID + 2,
+	TableColumns:          autoid.InformationSchemaDBID + 3,
+	tableColumnStatistics: autoid.InformationSchemaDBID + 4,
+	TableStatistics:       autoid.InformationSchemaDBID + 5,
+	TableCharacterSets:    autoid.InformationSchemaDBID + 6,
+	TableCollations:       autoid.InformationSchemaDBID + 7,
+	tableFiles:            autoid.InformationSchemaDBID + 8,
+	CatalogVal:            autoid.InformationSchemaDBID + 9,
+	TableProfiling:        autoid.InformationSchemaDBID + 10,
+	TablePartitions:       autoid.InformationSchemaDBID + 11,
+	TableKeyColumn:        autoid.InformationSchemaDBID + 12,
+	TableReferConst:       autoid.InformationSchemaDBID + 13,
+	// TableSessionVar:                         autoid.InformationSchemaDBID + 14,
```

Comment:
Why comment out this and remove the other three? (`tableGlobalStatus`,`tableGlobalVariables` and `tableSessionStatus`)

Reply:
I did this to document that `autoid.InformationSchemaDBID + 14` has been used for `TableSessionVar`. Removing this line would look like an unexplained gap. Looks like the same was done for `TablePlacementPolicy` on line 306.

I'm ok to to remove this line instead of commenting it out if you think that's better.

Reply:
Thanks for the explanation, please add a comment instead of why it is commented out, instead of removing it.